core/gpslocation.cpp

```diff
@@ -612,8 +612,8 @@ void GpsLocation::downloadFromServer()
 
 				struct gpsTracker gt;
 				gt.when = timestamp.toMSecsSinceEpoch() / 1000;
-				gt.latitude.udeg = latitude.toDouble() * 1000000;
-				gt.longitude.udeg = longitude.toDouble() * 1000000;
+				gt.latitude.udeg = lrint(latitude.toDouble() * 1000000);
+				gt.longitude.udeg = lrint(longitude.toDouble() * 1000000);
```

Comment:
I'm not sure I completely understand all the elements of that feature, but from what I understand I would say that, since fixes are identified by their fix timestamps (for create/delete/update operations) this should work ok. So in the case user re-download a fix it would be updated with new rounded value. @dirkhh Can you confirm or infirm this assumption?